app/controllers/planning_applications/validation/sitemaps_controller.rb

```diff
@@ -33,7 +33,7 @@ def update
         audit_action = @planning_application.boundary_geojson.blank? ? "created" : "updated"
 
         respond_to do |format|
-          if @planning_application.update(boundary_geojson: boundary_geojson_params[:boundary_geojson],
+          if @planning_application.update(boundary_geojson: JSON.parse(boundary_geojson_params[:boundary_geojson]),
```

Comment:
This looks great

Although, I think it would be great to include the `JSON.parse` in the setter's model, similar to `new_geojson=(value)` on `RedLineBoundaryChangeValidationRequest` like [this](https://github.com/unboxed/bops/blob/main/app/models/red_line_boundary_change_validation_request.rb#L27-L34)